src/annotator/integrations/pdf.js

```diff
@@ -100,12 +104,24 @@ export class PDFIntegration {
     this._reanchoringMaxWait = options.reanchoringMaxWait ?? 3000;
 
     /**
-     * A banner shown at the top of the PDF viewer warning the user if the PDF
-     * is not suitable for use with Hypothesis.
+     * Banners shown at the top of the PDF viewer.
```

Comment:
What's the intended behavior if both banners are applicable?

Reply:
You should see both banners. I don't know if JSTOR will serve us any PDFs that have no text at all, not even a front matter page, but there is content on JSTOR that is non-textual (eg. art, old hand-written letters that can't/haven't been OCR-ed).